tracer/test/snapshots/MongoDbTests.SubmitsTraces_packageVersion=2_2.SchemaV1.verified.txt

```diff
@@ -15,6 +15,8 @@
       runtime-id: Guid_2,
       span.kind: internal,
       version: 1.0.0,
+      _dd.git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbb,
```

Comment:
I don't get why the snapshots are changed with those changes

Reply:
These are due to the .NET 8 update. The .NET 8 SDK adds sourcelink by default to all apps. Which means we then extract them and add them to the tags (for source code integration support). 

They're added here, because this snapshot doesn't run by "default", only when you do the comprehensive testing. And due to space issues we _can't_ run comprehensive testing on more than one integration at a time. So I couldn't _necessarily_ update all the snapshots in the original .NET 8 PR.